lama-ali/script.js

```diff
@@ -0,0 +1,37 @@
+function Student(name, age, grade) {
+  this.name = name;
+  this.age = age;
+  let _grade = null;
+
+  const validGrade = (value) => value === "A" || value === "F";
+
+  this.introduce = function () {
+    console.log(
+      `Hi, my name is ${this.name}, I’m ${this.age} years old, and I’m in grade ${_grade}.`
+    );
+  };
+
+  this.getGrade = function () {
+    return _grade;
+  };
+
+  this.setGrade = function (value) {
```

Comment:
`setGrade` only job is to assign a value to `_grade` not to return anything, there should be a separate validator function that returns if the passed grade value is valid or not